src/main/webapp/gameBoard.html

```diff
@@ -54,7 +54,7 @@ <h5 id="users-name"> User Name </h5>
                     <p> User's Plant</p> 
                 </div>
                 <div class="column right">
-                    <h5 id="rank"> Your in Nth place! </h5>
+                    <h5 id="rank"></h5>
```

Comment:
You can probably get rid of this tag altogether now. I don't think we are using it for anything

Reply:
Got it, I took that out.